Rakefile

```diff
@@ -21,14 +23,17 @@ require 'bundler_api/gem_helper'
 $stdout.sync = true
 Thread.abort_on_exception = true
 
+RuboCop::RakeTask.new
+
 begin
   require 'rspec/core/rake_task'
 
   desc "Run specs"
   RSpec::Core::RakeTask.new(:spec) do |t|
     t.rspec_opts = %w(-fs --color)
-    #t.ruby_opts  = %w(-w)
+    Rake::Task['rubocop'].invoke
```

Comment:
No, it doesn't work. I've tried inside the block:

```ruby
  desc "Run specs"
  RSpec::Core::RakeTask.new(:spec) do |t|
    task :rspec => :rubocop
    t.rspec_opts = %w(--color)
  end
```
and outside:
```ruby
  desc "Run specs"
  RSpec::Core::RakeTask.new(:spec) do |t|
    t.rspec_opts = %w(--color)
  end
  task :rspec => :rubocop
```

But the only way it works is using like I did the first time:

```ruby
Rake::Task['rubocop'].invoke
```

And it works inside or outside the block, you choose. ;)